content/guides/ecommerce-connection/step3-reconcile-PIM-data-with-eCommerce-data.md

```diff
@@ -480,7 +480,7 @@ Your users will push you to have immediate synchronisation between Akeneo PIM an
 
 **Why?**
 * Because product data must be fully prepared (notion of completeness) before being synchronized with your eCommerce solution.
-* Because Akeneo PIM API does not yet have a push event system to determine whether data has been modified or deleted.
+* Because the Akeneo PIM does not yet have a push event system to determine whether data has been modified or deleted.
```

Comment:
To remove for the 5.0 ?

Reply:
I still need to review the api doc and update it to be 5.0 compliant. It will be in another PR.